frontend/src/scenes/billing/Plan.tsx

```diff
@@ -0,0 +1,104 @@
+import { Skeleton } from 'antd'
+import React, { useEffect, useState } from 'react'
+import { PlanInterface } from '~/types'
+import clsx from 'clsx'
+import { LemonButton, LemonDivider } from '@posthog/lemon-ui'
+import { IconLock } from 'lib/components/icons'
+
+export function Plan({
+    plan,
+    onSubscribe,
+    currentPlan,
+    canHideDetails = true,
+    primaryCallToAction = true,
+}: {
+    plan: PlanInterface
+    canHideDetails?: boolean
+    onSubscribe?: (plan: PlanInterface) => void
+    currentPlan?: boolean
+    primaryCallToAction?: boolean
+}): JSX.Element {
+    const [detail, setDetail] = useState('')
+    const [isDetailLoading, setIsDetailLoading] = useState(true)
+    const [showDetails, setShowDetails] = useState(false)
+
+    const loadPlanDetail = async (key: string): Promise<void> => {
+        const response = await fetch(`/api/plans/${key}/template/`)
+        if (response.ok) {
+            setDetail(await response.text())
+        }
+        setIsDetailLoading(false)
+    }
+
+    useEffect(() => {
+        loadPlanDetail(plan.key)
+    }, [plan.key])
+
+    return (
+        <div className="BillingPlan border rounded p-4">
```

Comment:
This doesn't fill the width of the page if the plan name is on the shorter side, `flex-grow: 1` would help:
<img width="1162" alt="Screen Shot 2022-09-02 at 11 12 48" src="https://user-images.githubusercontent.com/4550621/188111188-29fa36b2-7310-45b9-b3cc-774836e68e28.png">